### DISCUSSION TITLE
Folium backend shows nothing in gradio

### DISCUSSION THREAD

**Discussion by chaoer** [opengeos/leafmap#728] [Q&A] [answered]
I tried the Folium backend [demo ](https://leafmap.org/notebooks/66_gradio/) , but nothings shows. Is there anything I missed?
![image](https://github.com/opengeos/leafmap/assets/10883500/8c3ea58d-a87d-4efa-a54b-f1e7f52676a6)

**Comment by giswqs** [comment, marked as answer]
I have not used gradio for a while  now. Not sure if some recent gradio updates break the split map functionality. I recommend using Solara. See example at https://huggingface.co/spaces/giswqs/solara-maxar

https://github.com/opengeos/leafmap/assets/5016453/9bfa679e-531c-47ce-9dcd-16c7cbf16d84
